addons/web_editor/views/snippets.xml

```diff
@@ -380,6 +380,27 @@
     <div data-js="ImageOptimize"
          data-selector="img">
         <t t-call="web_editor.snippet_options_image_optimization_widgets"/>
+        <we-select string="Shape" data-name="shape">
+            <we-button data-shape="">None</we-button>
+            <we-button data-shape="web_editor/blob_1_solid_rd"  data-select-label="Blob 01">Blob 1</we-button>
+            <we-button data-shape="web_editor/blob_2_solid_str" data-select-label="Blob 02">Blob 2</we-button>
+            <we-button data-shape="web_editor/blob_3_solid_rd" data-select-label="Blob 03">Blob 3</we-button>
+            <we-button data-shape="web_editor/organic_1_line" data-select-label="Organic Line 01">Organic Line</we-button>
+            <we-button data-shape="web_editor/organic_2_pattern_dot" data-select-label="Organic Dot 01">Organic Dot</we-button>
+            <we-button data-shape="web_editor/organic_3_pattern_cross" data-select-label="Organic Cross 01">Organic Cross</we-button>
+            <we-button data-shape="web_editor/organic_4_pattern_caps" data-select-label="Organic Caps 01">Organic Caps</we-button>
+            <we-button data-shape="web_editor/oval_1_solid_rd">Oval 1</we-button>
+            <we-button data-shape="web_editor/oval_2_line">Oval 2</we-button>
+            <we-button data-shape="web_editor/oval_3_pattern_line">Oval 3</we-button>
+            <we-button data-shape="web_editor/fixed_random_blob">Multiple Colors</we-button>
+        </we-select>
+        <we-row string="Colors">
+            <we-colorpicker data-color="true" data-dependencies="shape" data-name="colors" data-color-id="0" />
+            <we-colorpicker data-color="true" data-dependencies="shape" data-name="colors" data-color-id="1" />
+            <we-colorpicker data-color="true" data-dependencies="shape" data-name="colors" data-color-id="2" />
+            <we-colorpicker data-color="true" data-dependencies="shape" data-name="colors" data-color-id="3" />
+            <we-colorpicker data-color="true" data-dependencies="shape" data-name="colors" data-color-id="4" />
```

Comment:
I used it to compute the widget visibility. As they all share the same behaviour I gave them the same name but I could incorporate the ID inside their name. It would just make the visibility code a little messier since I would have to split the widget name rather than parse params.colorId